tests/app/service/views/test_rest.py

```diff
@@ -261,3 +262,87 @@ def test_delete_service_not_exists(notify_api, notify_db, notify_db_session, sam
                 headers=[('Content-Type', 'application/json')])
             assert resp.status_code == 404
             assert Service.query.count() == 1
+
+
+def test_renew_token_should_return_token_when_service_does_not_have_a_valid_token(notify_api, notify_db,
+                                                                                  notify_db_session, sample_service):
+    with notify_api.test_request_context():
+        with notify_api.test_client() as client:
+            response = client.post(url_for('service.renew_token', service_id=sample_service.id),
+                                   headers=[('Content-Type', 'application/json')])
+            assert response.status_code == 201
+            assert response.get_data is not None
+            saved_token = Token.query.first()
+            assert saved_token.service_id == sample_service.id
+
+
+def test_renew_token_should_expire_the_old_token_and_create_a_new_token(notify_api, notify_db, notify_db_session,
```

Comment:
Can we get a sample_token py.test fixture created so we can easily add tokens to tests in the future.
It would also be useful for this test as well.